source/clear-linux/get-started/virtual-machine-install/virtualbox-cl-installer.rst

```diff
@@ -1,258 +1,357 @@
+#. A :guilabel:`choose a virtual optical disk file` browser window appears.
    Navigate to the extracted ISO file, select it, and click *Open*.
 
-   .. image:: ./figures/vbox/vbox-vm-settings-browse-ISO.png
-      :alt: Mounting an ISO in VirtualBox VM Settings  
-
-#. Click *OK* to exit the *VM Settings* menu and return to the main 
-   |VBM|.
+   .. figure:: figures/vbox/virtualbox-cl-installer-06.png
+      :scale: 100%
+      :alt: Mounting an ISO
 
+      Figure 6: Mounting an ISO
 
+#. Select :guilabel:`OK` to exit and return to the main |VBM|.
 
 Install |CL| using the installer
 ================================
 
-#. Start the VM from the |VBM| by selecting the |CL| VM and 
-   clicking *Start*
+#. In the |VBM|, select virtual machine you created, `ClearLinuxOS-VM`,
+   and click *Start*
 
-   .. image:: ./figures/vbox/vbox-start-vm.png
-      :alt: Starting a VirtualBox VM
+   .. figure:: figures/vbox/virtualbox-cl-installer-07.png
+      :scale: 100%
+      :alt: Start the installer
 
-#. A new window of the VM console will appear and boot into the |CL|
-   installer. Follow the steps in :ref:`bare-metal-install-desktop` to
-   install |CL| onto the VM virtual disk.
+      Figure 7: Start the installer
 
-.. note::
-   Do not choose a different kernel from the installer. **kernel-lts**, the 
-   Long Term Support (LTS) kernel is required for |VB| driver compatibility.
+#. When the boot manager appears, select :guilabel:`Clear Linux Installer`
+   and :guilabel:`Enter`.
 
+#. Follow the steps in :ref:`bare-metal-install-server` to
+   install |CL| onto the VM virtual disk. Note the following:
 
-#. After |CL| installation is complete, the VM will reboot and return to the 
-   |CL| installer. 
+   #. In :guilabel:`Configure Installation Media`,
+      select VBOX HARDDISK, and then select :guilabel:`Confirm`.
 
-.. note::
-   To release the mouse cursor from the VM console window, press the right Ctrl key on the keyboard.
+   #. In :guilabel:`Manage User`: Create an Administrative User.
+
+   #. Optional: If you want to add GNOME desktop, during installation, enter
+      :guilabel:`Advanced options` > `Select additional bundles`, and select
+      `desktop-autostart`.
+
+#. When |CL| installation is complete, select :guilabel:`Exit`.
+
+#. Power off the |CL| installer ISO and return to the |VBM|.
+
+   .. note::
 
+      To release the mouse cursor from the VM console window, press the right
+      :kbd:`Ctrl` key on the keyboard.
 
 Unmount the installation ISO
 ============================
 
 The |CL| installer ISO needs to be unmounted to allow the VM to boot from the
-virtual hard disk, which |CL| has been installed to.
+virtual hard disk where |CL| has been installed.
 
+#. In case the |CL| VM is not off, select :guilabel:`Machine>ACPI Shutdown`.
 
-#. Power off the |CL| VM.
-
-   .. image:: ./figures/vbox/vbox-shutdown-vm.png
+   .. figure:: figures/vbox/virtualbox-cl-installer-08.png
```

Comment:
The top of this VM window shows the desktop version which may confuse people.

Reply:
The point of this screenshot is simply to show where `ACPI Shutdown` is on the menu. 
Where in Fig. 8 do you think it shows the "desktop" version?

Reply:
Just the top GNOME shell bar with the date/time.